README.md

```diff
@@ -0,0 +1,24 @@
+# ros2cli
+
+This repository contains the source code for ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
+
+## Usage
+
+Run `ros2 --help` to see all available verbs.
+
+Run `ros2 <verb> --help` for more information on individual command usage.
+
+Read [Introspection with command line tools](https://index.ros.org/doc/ros2/Tutorials/Introspection-with-command-line-tools/) on ROS Index for more information and an example.
+
+### Cheat Sheet
+
+This [cheat sheet](https://github.com/artivis/ros2_cheats_sheet/blob/master/cli/cli_cheats_sheet.pdf) provides examples for verbs and their sub-verbs (some of which rely on the [ROS 2 demos package](https://github.com/ros2/demos)).
```

Comment:
Nice! I didn't know about it.

Reply:
Do someone here has write access on that repo? If not, it would be good to have a local copy of that here (if it can be copied), or generate something similar locally.

Reply:
I think that @artivis would probably be open to collaborating on it, if we wanted to do PRs there.  I don't know where we would bring it in locally?

Reply:
I'm definitely open to collaboration/PRs or even move it as you see fit !
The Latex source is hosted on the repo after all ;)

Reply:
And it's already Apache-2, how about that :)